config/cageConfiguration.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" ?>
+<cageConfiguration>
+	<cageStructure>
+		<height>200</height>
+		<radius>60</radius>
+	</cageStructure>
+	<muscleUnits>
```

Comment:
position of the motor can be interpreted as an attachment point 0 of each tendon